packages/store/src/adapter/base.ts

```diff
@@ -64,6 +66,10 @@ export abstract class BaseAdapter<AdapterTarget = unknown> {
   abstract toSliceSnapshot(
     payload: ToSliceSnapshotPayload<AdapterTarget>
   ): Promise<SliceSnapshot>;
+
+  applyConfigs(configs: Map<string, string>) {
+    this.configs = new Map([...configs]);
```

Comment:
Do we only allow string as config value?

Reply:
For now, that is true.
It can be any though.